.gitignore

```diff
@@ -1,6 +1,10 @@
 dist
 testdata/simple*/docs
 cover.out
+coverage/*
+.nyc_output/*
+
+*.orig
```

Comment:
These don't need to add in `.gitignore` file.

Reply:
Why not? 
Won't harm anything to block these, as these files are automatically generated and not needed.

Reply:
These files won't harm anything as you said. But for examples, .orig files are auto generated by git mergetools and these don't need to ignore for swag proejct actually.

Reply:
okay, I'll remove it again. That extension to gitignore made it easier for me not to check in those files by accident.